charts/doit-eks-lens/templates/doit-otelcol.yaml

```diff
@@ -394,6 +394,10 @@ spec:
               fieldPath: metadata.namespace
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: "cluster.local"
+        {{- range .Values.collector.otelcol.env }}
```

Comment:
One of these parts doesn't look right - it's currently `.Values.collector.env`, so either this needs to change, or the indentation in `values.yaml` is wrong.